### PR TITLE
Rjm/nbe stuff

### PR DESCRIPTION
**Pull request by rjmac**

### CODE REVIEW

File: src/main/java/com/socrata/datasync/ui/PortJobTab.java [hunk 1]
```diff
@@ -146,6 +152,15 @@ public PortJobTab(PortJob job, JFrame containingFrame) {
         destinationSetIDTextFieldContainer.add(openSinkDatasetButton);
         jobPanel.add(destinationSetIDTextFieldContainer);
 
+        if(useUseNewBackend) {
+            jobPanel.add(UIUtility.generateLabelWithHelpBubble("Use new checkbox", NBE_TIP_TEXT, HELP_ICON_TOP_PADDING));
```
Comment (catstavi) [inline src/main/java/com/socrata/datasync/ui/PortJobTab.java]: I'm having trouble picturing where this text shows up in a way that makes sense.

Comment (rjmac) [reply src/main/java/com/socrata/datasync/ui/PortJobTab.java]: In the main port job panel, right below the "destination dataset id" field

File: src/main/java/com/socrata/datasync/ui/PortJobTab.java [hunk 2]
```diff
@@ -76,6 +79,9 @@
 
     // build Container with all tab components and load data into form
     public PortJobTab(PortJob job, JFrame containingFrame) {
+        boolean useUseNewBackend = System.getenv("SOCRATA_SHOW_NBE_CHECKBOX") != null;
```
Comment (catstavi) [inline src/main/java/com/socrata/datasync/ui/PortJobTab.java]: Seekrit Kohd

**Comment by catstavi** [comment]
This all looks reasonable to me. We should probably do a thorough demo.

File: src/main/java/com/socrata/datasync/ui/SimpleIntegrationWizard.java [hunk 3]
```diff
@@ -291,7 +294,9 @@ protected void done() {
             runJobNowButton.setEnabled(true);
 
             // show popup with returned status
-            if(jobStatus.isError()) {
+            if(jobStatus == null) {
+                System.out.println("null jobStatus?!");
```
Comment (catstavi) [inline src/main/java/com/socrata/datasync/ui/SimpleIntegrationWizard.java]: was this a debugging thing? should we have a more serious message?

File: src/main/java/com/socrata/datasync/ui/SimpleIntegrationWizard.java [hunk 4]
```diff
@@ -1,907 +1,965 @@
...
+
+    private class NewMetadataJobListener implements ActionListener {
+        public void actionPerformed(ActionEvent e) {
+            addJobTab(new MetadataJob());
+            jobTabsPane.setSelectedIndex(jobTabsPane.getTabCount() - 1);
+        }
+    }
+
+    /**
+     * Listen for acti
```
Comment (catstavi) [inline src/main/java/com/socrata/datasync/ui/SimpleIntegrationWizard.java]: What's going on here?

Comment (catstavi) [inline src/main/java/com/socrata/datasync/ui/SimpleIntegrationWizard.java]: comment code?

**Pull request merged**